cabal.project

```diff
@@ -1,4 +1 @@
 packages: .
-debug-info: 3
-executable-stripping: False
-library-stripping: False
```

Comment:
I think I got confused about this. We probably need to avoid stripping the executable don't we?

We should pass `-g3` directly to the test executable with GHC options though.

Reply:
The example still works for me with these removed (making sure to clean `dist-newstyle` first) on Mac, can you reproduce this on linux?